themes/makedeb/assets/scss/variables.scss

```diff
@@ -10,3 +10,41 @@ $silver: #f7f7f7;
 $inactive-link: #919090;
 $header-bg-color: #f2f2f2;
 $header-hg-color: #e6e5e3;
+
+.dark .themeHeaderColor {
+  background-color: #1e1e1e;
+}
+.dark .themeTextColor {
+  color: #fff;
+}
+.dark .themeHeaderHrColor {
+  color: #111111;
+  background-color: #111111;
+}
+.dark .themeBackgroundColor {
+  color: #fff;
+  background-color: #2d2d2d;
+  border-color: #464646;
+}
+.dark .themeFactsColor {
+  color: #fff;
+  background-color: #272727;
+}
+.dark .themeCodeColor {
+  color: #fff;
+  background-color: #202020;
+}
+.dark .themeClipboardCheckedColor {
+  color: #40a85c;
+  background-color: #202020;
+}
+.dark .themePostContentDateColor {
+    color: #bfbfbf;
+}
+.dark .themePostHrColor {
+    background-color: #6a6a6a;
+}
+.dark .themeCardsColor {
+    background-color: #262626;
+    color: #fff;
+}
```

Comment:
I just realized this, but could `.light .theme*` variants also be created? The places where these are used already have a `background-color` defined manually (such as in `scss/post.scss:5`), and if they were defined in a `.light .theme*` block in here (and the the `background-color`/`color`/etc. calls removed in the other places) it would make the code more consistent.